app/views/landing_page/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="landing-page">
+  <h1>Lifestyle Budget App</h1>
+  <div class="lower-part">
+    <span class=""><%= link_to "LOG IN", new_user_session_path %></span>
+    <span class=""><%= link_to "SIGN UP", new_user_registration_path %></span>
+  </div>
+</div>
```

Comment:
- [ ] Please make sure to follow the design guidelines:

|Template|Your design|
|---|---|
| ![image](https://github.com/joyapisi/budget-app/assets/77069011/164e9688-59ec-44ca-9f5d-d19b5626ce1e) | ![image](https://github.com/joyapisi/budget-app/assets/77069011/00a52748-bd8e-40ef-801d-e96c849c0836) |

According to the requirements:

![image](https://github.com/joyapisi/budget-app/assets/77069011/8b34ab31-bd7c-4ae5-b64b-85b7808a520f)

Please consider working on this! 🤓 🔧